zbus/src/fdo.rs

```diff
@@ -17,8 +17,8 @@ use zvariant::{
 };
 
 use crate::{
-    dbus_interface, dbus_proxy, message::Header, object_server::SignalContext, DBusError, Guid,
-    ObjectServer,
+    dbus_interface, dbus_proxy, message::Header, object_server::SignalContext, proxy::ProxyImpl,
+    DBusError, Guid, ObjectServer,
```

Comment:
That's actually really smart. I've never seen that pattern.

I actually thought about how I could circumvent existing code from breaking because they have to import the trait, but couldn't come up with a nice solution.